code/src/components/UncompletedTodosCount.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+const UncompletedTodosCount = () => {
+  const todosComplete = useSelector(state => state.todos.items)
+  const uncompletedTodos = todosComplete.filter(todo => todo.isComplete === false).length
```

Comment:
As you're not using the todosComplete here, you could just get the length of the uncompleted todos from the store in one line:
const uncompletedTodos = useSelector(state => state.todos.items.filter(item=> !item.isComplete).length)